Editorial review: Document Wasm bulk memory operations (#44296)
mdn/content: files/en-us/webassembly/reference/elem/drop/index.md

---
title: "drop: Wasm elem instruction"
short-title: drop
slug: WebAssembly/Reference/Elem/drop
page-type: webassembly-instruction
browser-compat: webassembly.elem.drop
sidebar: webassemblysidebar
---

> The **`elem.drop`** [elem instruction](/en-US/docs/WebAssembly/Reference/Elem) discards the data contained by a [passive](/en-US/docs/WebAssembly/Reference/Definitions/elem#passive_form) `elem` segment, freeing up its memory, after the associated `table` has been initialized via [`table.init`](/en-US/docs/WebAssembly/Reference/Table/init).

Reviewer: Nitpick: A passive element segment doesn't really have an "associated table". Active ones do, but passive ones don't.

```suggestion
The **`elem.drop`** [elem instruction](/en-US/docs/WebAssembly/Reference/Elem) discards the data contained by a [passive](/en-US/docs/WebAssembly/Reference/Definitions/elem#passive_form) `elem` segment, freeing up its memory, after being used in a [`table.init`](/en-US/docs/WebAssembly/Reference/Table/init).
```

Author: OK, makes sense. Updated here, and on the parent page.
